mono/mini/interp/interp.c

```diff
@@ -3472,10 +3472,10 @@ interp_exec_method (InterpFrame *frame, ThreadContext *context, FrameClauseArgs
 			++ip; 
 			MINT_IN_BREAK;
 		MINT_IN_CASE(MINT_DUP_VT) {
-			int i32 = READ32 (ip + 1);
```

Comment:
Won't this break some code which used to work ?

Reply:
Even before this change, some opcodes support 32bit sized value types while others only 16bit, so it's not like we really supported it. This change just doubles down on having all sizes fit into 16bit. If deemed necessary in the future, we can easily add some slow path opcodes, accepting the full size, similarly with the local offsets, which are also only 16bits